plutarch-extra/plutarch-extra.cabal

```diff
@@ -0,0 +1,88 @@
+cabal-version: 2.4
+name:          plutarch-extra
+version:       1.1.0
+
+flag development
```

Comment:
I just replicated it analogously to the other `*.cabal` files, I can remove it for now but I figured I'd leave it in for good measure, in case we need it at some point.

Reply:
I forget how exactly cabal flags work. If some function in `plutarch-extra` uses tracing functions within - and you want them to actually trace, is passing `+development` to `package plutarch` enough?

Reply:
/cc @V0ldek

Reply:
You don't need this.

Reply:
copied.